pkgs/http_client_conformance_tests/lib/http_client_conformance_tests.dart

```diff
@@ -22,6 +22,12 @@ export 'src/response_headers_tests.dart' show testResponseHeaders;
 
 /// Runs the entire test suite against the given [Client].
 ///
+/// If [packageRoot] is set then it will be used as the filesystem root
```

Comment:
Do we need a publish of `package:test`? Would that simplify this?

Reply:
Could we pass the `package:` URI without resolving it since you and @jakemac53 fixed the handling of `package:`?

Reply:
We could but that would only change one line in the `startServer` code.

Reply:
I don't know where I went wrong before. I merged in your change. Thanks and sorry for messing this up (somehow).